tools/generate_definitions.py

```
@@ -1,376 +1,288 @@
-)
-# name-to-value map - needed for SField processing
-type_map = {x[0]: x[1] for x in type_hits}
 
-
-def _is_vl_encoded(t: str) -> str:
-    if t == "VL" or t == "ACCOUNT" or t == "VECTOR256":
-        return "true"
-    return "false"
+def _exec(cmd: str) -> str:
+    """Run a shell command and return its stripped stdout."""
+    result = subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)